.gitignore

```diff
@@ -1,3 +1,4 @@
 /nbproject/private/
-*.ttf
-*.otf
\ No newline at end of file
+/assets/fonts/*.ttf
```

Comment:
SInce you're also adding a .gitignore file inside the /assets/font/ folder, I don't think these are necessary. But that's probably fine.